docs/credits.md

```diff
@@ -1294,25 +1308,26 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | Nancy.Serialization.JsonNet                       | Andreas Håkansson, Steven Robbins and contributors                        | [http://nancyfx.org/](http://nancyfx.org/)                                                                                                                                                                                  | [MIT](https://github.com/NancyFx/Nancy/blob/master/license.txt)                                                     |
 | nanoid                                            | Andrey Sitnik                                                             | [https://github.com/ai/nanoid](https://github.com/ai/nanoid)                                                                                                                                                                | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | nanomatch                                         | Jon Schlinkert                                                            | [https://github.com/micromatch/nanomatch](https://github.com/micromatch/nanomatch)                                                                                                                                          | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | natural-compare                                   | Lauri Rooden                                                              | [https://github.com/litejs/natural-compare-lite](https://github.com/litejs/natural-compare-lite)                                                                                                                            | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | NCrontab.Advanced                                 | jcoutch                                                                   | [https://github.com/jcoutch/NCrontab-Advanced](https://github.com/jcoutch/NCrontab-Advanced)                                                                                                                                | [Apache-2.0](https://github.com/jcoutch/NCrontab-Advanced/blob/master/LICENSE)                                      |
+| NCrontab.Advanced                                 | jcoutch                                                                   | [https://github.com/jcoutch/NCrontab-Advanced](https://github.com/jcoutch/NCrontab-Advanced)                                                                                                                                | [Undefined]()                                                                                                       |
```

Comment:
This is Apache 2.0 - @N-lson I wonder why it didn't pick this up properly?

Reply:
This was being picked up twice and we only set the license info if it matched package name + author - fix here https://github.com/OctopusDeploy/DependencyScanner/pull/54